package.json

```diff
@@ -69,11 +69,14 @@
     "braces": "^3.0.3",
     "css-what": "^5.0.1",
     "micromatch": "^4.0.8",
-    "nx": "^19.1.4",
+    "axios": "^1.8.2",
```

Comment:
It fixes the Trivy vulnerability warning for axios:
[CVE-2025-27152 | Server-Side Request Forgery (SSRF)](https://avd.aquasec.com/nvd/2025/cve-2025-27152/)

This `axios` is part of the `nx` package, but even with the latest version 19, the axios package is still in a vulnerable version.